modules/www/manifests/code_submitter.pp

```diff
@@ -23,6 +22,14 @@
     require => Vcsrepo[$root_dir],
   }
 
+  file { '/etc/sr/code-submitter-credentials.yaml':
+    ensure => present,
+    owner => 'wwwcontent',
+    group => 'apache',
+    mode => '440',
+    source => "/srv/secrets/code-submitter-credentials.yaml",
```

Comment:
nit: since there aren't any variables here, I think this can have single quotes.

Reply:
This was just copied verbatim from https://github.com/srobo/server-puppet/blob/master/modules/www/manifests/ide.pp#L309

Reply:
9a22abb

Reply:
```suggestion
    source => '/srv/secrets/code-submitter-credentials.yaml',
```

Reply:
I think this still applies?

Reply:
928bc1a